projects/laji/src/app/shared-modules/laji-form/laji-form/laji-form.component.ts

```diff
@@ -124,28 +125,29 @@ export class LajiFormComponent implements OnDestroy, OnChanges, AfterViewInit {
   }
 
   popErrorListIfNeeded() {
-    this.lajiFormWrapper.app.refs.lajiform.popErrorListIfNeeded();
+    this.lajiFormWrapper.lajiForm.popErrorListIfNeeded();
   }
 
   private mount() {
     if (!this.formData?.formData) {
       return;
     }
-    import('laji-form').then((formPackage) => {
+    combineLatest(
+      import('laji-form'),
+      import('laji-form/lib/themes/bs3'),
+      this.userService.getUserSetting<any>(this.settingsKey).pipe(
+        concatMap(settings => this.userService.getUserSetting<any>(GLOBAL_SETTINGS).pipe(
+          map(globalSettings => ({...globalSettings, ...settings}))
+        )),
+        take(1)
+      ),
+      this.userService.getProfile().pipe(map(profile => profile.settings?.defaultMediaMetadata))
```

Comment:
Since there was no take(1) in this I added one to getProfile in UserService (since that looked like a method that should only return one result) otherwise this subscription would not complete (not an issue in here, but could become issue somewhere else)